testsuite/MDAnalysisTests/formats/test_libdcd.py

```diff
@@ -84,7 +86,7 @@ def test_raise_not_existing(self):
     def test_n_atoms(self):
         assert_equal(self.dcdfile.n_atoms, 3341)
 
-    @raises(IOError)
+    @raises(RuntimeError)
```

Comment:
I've think in the xdr wrapper I changed everything to raise an IO error. I think that would be a  good idea here to.